examples/Tools/ChooseRadioAccessTechnology/ChooseRadioAccessTechnology.ino

```diff
@@ -0,0 +1,126 @@
+
+/*
+  Radio Access Technology selection for Arduino MKR GSM 1400
+  This sketch allows you to select your preferred Radio Access
+  Technology (RAT).
+  You can choose among 2G, 3G(UMTS) or a combination of both.
+  Selecting JUST ONE technology will speed up the modem's network
+  registration A LOT!
+  The chosen configuration will be saved to modem's internal memory
+  and will be preserved through MKR GSM 1400 sketch uploads.
```

Comment:
The spacing seems different from the original here: https://github.com/arduino-libraries/MKRNB/blob/master/examples/Tools/ChooseRadioAccessTechnology/ChooseRadioAccessTechnology.ino

Reply:
changed!